src/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Calculator from './components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] In your `index.js` you  should render your `<App/>` component and then your `<Calculator/>` should be rendered inside your `<App/>` component, Kindly fix that to make your code look more awesome and maintanable